catalyst_voices_packages/catalyst_cardano_serialization/lib/src/rbac/x509_metadata_envelope.dart

```diff
@@ -266,15 +268,15 @@ final class X509MetadataEnvelope<T> extends Equatable {
     );
   }
 
-  static List<int>? _compressBrotli(List<int> bytes) {
+  static Future<List<int>?> _compressBrotli(List<int> bytes) async {
     try {
       return CatalystCompression.instance.brotli.compress(bytes);
     } on CompressionNotSupportedException {
       return null;
     }
```

Comment:
The future must be awaited inside the try-catch otherwise the exception is not caught by this try-catch. We can combine it with return like this:

```suggestion
    try {
      return await CatalystCompression.instance.brotli.compress(bytes);
    } on CompressionNotSupportedException {
      return null;
    }
```